coalib/output/ConsoleInteraction.py

```diff
@@ -264,6 +266,15 @@ def print_result(console_printer,
                                     if not isinstance(action, ShowPatchAction))
             else:
                 print_diffs_info(result.diffs, console_printer)
+        print_aspect_action = PrintAspectAction()
+        if print_aspect_action.is_applicable(
+                result, file_dict, file_diff_dict) is True:
+            print_aspect_action.apply_from_section(result,
+                                                   file_dict,
+                                                   file_diff_dict,
+                                                   section)
+            cli_actions = tuple(action for action in cli_actions
+                                if not isinstance(action, PrintAspectAction))
```

Comment:
This new `PrintAspectAction` block is pretty similar to the existing `ShowPatchAction` block above. We should think about some abstraction here...